pkg/spec/createconfig.go

```diff
@@ -287,6 +287,11 @@ func (c *CreateConfig) getContainerCreateOptions(runtime *libpod.Runtime, pod *l
 		options = append(options, libpod.WithLogPath(logPath))
 	}
 
+	logTag := getLoggingTag(c.LogDriverOpt)
```

Comment:
"Logging" vs "Log" inconsistency

Reply:
I've used the same convention used above for `logPath := getLoggingPath(c.LogDriverOpt)`